docs/processes-v2/flows.md

```diff
@@ -458,8 +458,8 @@ flows:
       in:
         msg: "${item.key} - ${item.value}"
       withItems:
-        a: "Hello"
```

Comment:
I believe the `withItems` part is correct in this example. It is using `withItems` with a dict (Map) object. It iterates over the objects keys and is an equivalent of
```json
"withItems": {
  "a": "Hello",
  "b": "world"
}
```

Example output:
```
> cat concord.yml
flows:
  default:
    - task: log
      in:
        msg: "${item.key} - ${item.value}"
      withItems:
        a: "Hello"
        b: "world"

> concord run
Starting...
13:14:15.308 [main] a - Hello
13:14:15.310 [main] b - world
...done!
```

Although we should change `call: log` to `task: log` to make the example runnable.

Reply:
Ok. Maybe you can change that. Thanks.